mne_bids/dig.py

```diff
@@ -543,6 +546,104 @@ def _read_dig_bids(electrodes_fpath, coordsystem_fpath,
     # put back in unknown for unknown coordinate frame
     if coord_frame == 'unknown':
         for ch in raw.info['chs']:
-            ch['coord_frame'] = _str_to_frame['unknown']
+            ch['coord_frame'] = MNE_STR_TO_FRAME['unknown']
         for d in raw.info['dig']:
-            d['coord_frame'] = _str_to_frame['unknown']
+            d['coord_frame'] = MNE_STR_TO_FRAME['unknown']
+
+
+@verbose
+def template_to_head(raw, space, coord_frame='auto', unit='auto',
+                     verbose=None):
+    """Transform a BIDS standard template montage to the head coordinate frame.
+
+    Parameters
+    ----------
+    raw : mne.io.Raw
+        The MNE-Python Raw object with a montage in one of the BIDS standard
+        template coordinate frames (modified in place).
```

Comment:
Passing an `Info` or even simply a `DigMontage` should suffice to get the `trans`, right? I think we should support this… or do you think it doesn't make sense, because usually the user will have a `Raw` (or `Epochs` or …) that they want to "convert", and they're not looking for **just** a `trans`?

Reply:
I tried it with the montage originally but it was really cumbersome because you ended up getting, setting and then re-getting the montage in the examples. I'll change it to `info` though.

Reply:
I tried to use info but I forgot it doesn't have the `ContainsMixin` so you can't `get_montage` although you can set it. @larsoner or @agramfort, is there a reason `info` can't `get_montage`? I'd be happy to add it in a quick PR if not but I assume so.

Reply:
Or maybe `get_montage` should moved from the `ContainsMixin` to the `MontageMixin`, I'm not sure if that would break everything though

Reply:
Ok I opened a PR and since there is about to be a release, this will immediately work with the stable branch of MNE (after the release). Since it's a new function, it won't break for anyone but will be a great reminder to upgrade to 1.0.